### ISSUE TITLE
no template for config

### ISSUE THREAD

**Issue by flytigerboy** [ochinchina/supervisord#25]
There is no template for all configs.Must read the code to find them.

**Comment by radutopala** [comment]
@flytigerboy yeah, maybe a `supervisord init` subcommand to generate an initial config, would be nice to have.

**Comment by ochinchina** [comment]
subcommand init is added to initialize a template for supervisord configuration.
